ckan/tests/controllers/test_admin.py

```diff
@@ -233,10 +228,10 @@ def test_trash_view_sysadmin(self, app, sysadmin_env):
         # On the purge page
         assert "purge-all" in trash_response
 
-    def test_trash_no_datasets(self, app, sysadmin_env):
+    def test_trash_no_datasets(self, app, sysadmin_env, package_factory):
```

Comment:
Can we call it `dataset_factory`? :pray:

Reply:
Yes, it's possible.

I used "package" because I want to highlight that this dataset is not the only possible package type. In their own projects, people should create sub_factories for creating package factories with the non-dataset type.

And for dataset type, if the developer customizes it, then he can register a specific `dataset_factory` fixture, with the package[type=dataset] that has set of fields sensible for this particular portal's dataset. While `package_factory` is still available if he needs something more low-level